curriculum/challenges/english/07-scientific-computing-with-python/learn-interfaces-by-building-an-equation-solver/662fab95a3f903a7aa028550.md

```diff
@@ -0,0 +1,71 @@
+---
+id: 662fab95a3f903a7aa028550
+title: Step 11
+challengeType: 20
+dashedName: step-11
+---
+
+# --description--
+
+A property is a method that can be accessed as an attribute. The `property` decorator is used for this purpose:
+
+```py
+class Fish:
+    @property
+    def swim(self):
+        return 'Swimming!'
+
+trout = Fish()
+trout.swim # Swimming!
+```
+
+In the example above, the `swim` property of `trout` is accessed throught `trout.swim` instead of `trout.swim()`, as it would be for a regular method.
```

Comment:
```suggestion
In the example above, the `swim` property of `trout` is accessed through `trout.swim` instead of `trout.swim()`, as it would be for a regular method.
```